cvat/apps/quality_control/default_settings.py

```diff
@@ -6,3 +6,6 @@
 
 QUALITY_CHECK_JOB_DELAY = int(os.getenv("CVAT_QUALITY_CHECK_JOB_DELAY", 15 * 60))
 "The delay before the next quality check job is queued, in seconds"
+
+PROJECT_QUALITY_CHECK_JOB_DELAY = int(os.getenv("CVAT_PROJECT_QUALITY_CHECK_JOB_DELAY", 5 * 60))
```

Comment:
Project reporting is executed after the task reports. It's significantly cheaper to compute, unlike task reports. In case of projects, this delay is mainly to accumulate several task reports and not compute a new report on each task update. In the case of tasks, it's mainly to reduce server load because of hard computations.

Reply:
Okay. In that case, I suggest that you update the documentation string to mention that the delay starts after task reports.

Also, you should update the docstring for the preceding setting to note that it applies to tasks (I assume the setting itself cannot be renamed due to backwards compatibility).

Reply:
Added

Reply:
Not quite - you added what the delay is before, but you didn't add what it's after.

Reply:
It's computed after task quality reports, but not only after them, because sometimes there are no tasks in the projects. I don't see how this information would be useful here.